server/routes/test_routes.ts

```diff
@@ -0,0 +1,55 @@
+import { IRouter, IClusterClient, SessionStorageFactory } from "../../../../src/core/server";
+import { SecuritySessionCookie } from "../session/security_cookie";
+import { schema } from '@kbn/config-schema';
+import { User } from "../auth/user";
+import { SecurityClient } from "../backend/opendistro_security_client";
+import { CoreSetup } from "../../../../src/core/server";
+
+export function defineTestRoutes(router: IRouter,
+  securityConfigClient: IClusterClient,
+  sessionStorageFactory: SessionStorageFactory<SecuritySessionCookie>,
+  core: CoreSetup) {
+
+  router.get({
+    path: `/test/login`,
```

Comment:
is this for debug/test in development, could we disable it in production build?

Reply:
yeah, this is for testing purpose, as a replacement of the login page, will remove it after the login page is ready. I can add a TODO comment here